config/images/images.yaml

```diff
@@ -164,6 +164,18 @@ images:
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/opentelemetry-operator/opentelemetry-operator
   tags:
   - v0.127.0
+- source: persesdev/perses
+  destination: europe-docker.pkg.dev/gardener-project/releases/3rd/persesdev/perses
+  tags:
+  - v0.50.3
```

Comment:
I see gardener is using only version v0.51.0, why do you replicate older unused version?